src/ASM/ASMs2D.C

```diff
@@ -2630,6 +2648,12 @@ bool ASMs2D::evalSolution (Matrix& sField, const IntegrandBase& integrand,
   // Fetch nodal (control point) coordinates
   Matrix Xnod, Xtmp;
   this->getNodalCoordinates(Xnod);
+  if (integrand.getIntegrandType() & Integrand::UPDATED_NODES)
+  {
+    Vectors& eV = const_cast<IntegrandBase&>(integrand).getSolutions();
```

Comment:
isn't the const_cast a bit .. icky ?

Reply:
it is indeed, but I did not see any better solution, since the integrand argument is const

Reply:
The eV array is updated by the ASMbase::deformedConfig method, so a const overload of getSolutions will not work. Think therefore we leave it like it is, at least for now